addons/longhorn/1.2.2/install.sh

```diff
@@ -1,12 +1,23 @@
+# shellcheck disable=SC2148
+
 DID_MIGRATE_ROOK_PVCS=
 
 function longhorn_pre_init() {
+    local re='^[0-9]+$'
+
     if [ -z "$LONGHORN_UI_BIND_PORT" ]; then
         LONGHORN_UI_BIND_PORT="30880"
     fi
     if [ -z "$LONGHORN_UI_REPLICA_COUNT" ]; then
         LONGHORN_UI_REPLICA_COUNT="0"
     fi
+    # Error check that someone didn't add the percent sign to this integer
+    if [ -n "$LONGHORN_STORAGE_OVER_PROVISIONING_PERCENTAGE" ] && ! [[ $LONGHORN_STORAGE_OVER_PROVISIONING_PERCENTAGE =~ $re ]]; then
+        bail "You entered ${LONGHORN_STORAGE_OVER_PROVISIONING_PERCENTAGE} for LONGHORN_STORAGE_OVER_PROVISIONING_PERCENTAGE, but it must be an integer. e.g. 200"
+    fi
+    if [ -z "$LONGHORN_STORAGE_OVER_PROVISIONING_PERCENTAGE" ]; then
+        LONGHORN_STORAGE_OVER_PROVISIONING_PERCENTAGE="200"
```

Comment:
Just curious, can you explain to me why 200 is a sane default?

Reply:
Mostly just that's what the default from Longhorn is and I didn't want to change the existing behavior. The theory is that most workloads don't use up there volumes so it's fine to over-provision.  Some more discussion here: https://longhorn.io/docs/1.2.3/references/settings/#storage-over-provisioning-percentage